docs/README.md

```diff
@@ -24,101 +24,138 @@ kubectl -n gangway create secret generic gangway-key \
   --from-literal=sesssionkey=$(openssl rand -base64 32)
 ```
 
-## Docker image
+## Detailed Instructions
 
-A recent release of Gangway is available at
+The following guide is a more detailed review of how to get Gangway and other components configured.
+It's just an example of how a fully functional system can be constructed and components used may vary depending on the specific environment.
+
+It's also important to note that this guide does not include an Auth provider, that comes in the next section:
+
+We will use the following components:
+
+- [gangway](https://github.com/heptiolabs/gangway): OIDC client application
+- [contour](https://github.com/heptio/contour): Kubernetes Ingress controller
+- [cert-manager](https://github.com/jetstack/cert-manager): Controller for managing TLS certificates with Let's Encrypt.
+
+### Cert-Manager
 
+Run:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/v0.4.1/contrib/manifests/cert-manager/with-rbac.yaml
 ```
-gcr.io/heptio-images/gangway:<tag>
+
+Apply the following config to create the staging and production `ClusterIssuer` making sure to update the `email` field below with your email address:
+
+```sh
+cat <<EOF | kubectl create -f - 
```

Comment:
This config also lives in docs/yaml/cert-manager/... Should we dedupe?

Reply:
yeah I can delete from yaml, was trying to get a better way to not edit files, then apply, etc. Maybe we need helm? =)

Reply:
Haha yeah.